packages/page_object/analysis_options.yaml

```diff
@@ -0,0 +1,122 @@
+# pedantic has a strong foundation, but right now they only enable a small set of lints
```

Comment:
minor (not for this PR), but I wonder if we can move the shared analysis file to the root of the repo and reference it via ../../

then we don't have to duplicate per package.

can be a future optimization though

Reply:
Ya this is a good idea.

Reply:
#92